src/plugins/service-worker/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+const buildPaths = require('mwp-cli/src/commands/buildCommands/config/paths');
```

Comment:
what do you mean by 'interdependencies'? Currently there is a one-way dependency between mwp and mwp-cli, which seems reasonable to me - the platform depends on the platform builder for build config info.